package.json

```diff
@@ -37,8 +37,15 @@
     ".": {
       "types": "./dist/src/index.d.ts",
       "import": "./dist/src/index.js"
+    },
+    "./alloc-unsafe.js": {
+      "browser": "./dist/src/alloc-unsafe-browser.js",
+      "default": "./dist/src/alloc-unsafe.js"
     }
   },
+  "browser": {
+    "./dist/src/alloc-unsafe.js": "./dist/src/alloc-unsafe-browser.js"
```

Comment:
Do we need this as well as the exports map entry?  As I understand it, everything should just use the exports map.

Reply:
I also believe the exports map should be enough.

I included this line because the other js-libp2p projects that uses browser specific files **only** use this. No exports map.
Maybe there are some build tools used by the libp2p community that support this browser field but not exports? Happy to remove, just let me know.

Reply:
Ah, right, my mistake - we need the `"browser"` field so tools like esbuild/webpack/etc can override the internal import of the `alloc-unsafe.js` file when creating a bundle.

We don't need the addition to the exports map as that would only be used if someone did something like:

```js
import allocUnsafe from '@libp2p/mplex/alloc-unsafe'
```

Have pushed a new commit to the PR with this change.